### ISSUE TITLE
Error in Linux version 0.1.0

### ISSUE THREAD

**Issue by dante-el-gamer** [dante-el-gamer/management-of-class-positions#1]
When opening it on **Linux,** in my case, **the error occurs due to a missing Linux accessibility/appearance library (xapp-gtk3-module) and the video driver failing to start basic graphics acceleration (EGL). Therefore, the application displays nothing on Linux,** and there is no other option but to close the application. The error and the complete log are below.

```bash
YourName@OS:/$ cd /the/file/path
Gtk-Message: 19:11:42.303: Failed to load module "xapp-gtk3-module"
Gtk-Message: 19:11:42.544: Failed to load module "xapp-gtk3-module"
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...
```
